__tests__/app.test.js

```diff
@@ -65,6 +64,34 @@ describe('GET / api/reviews', () => {
         });
       });
   });
+
+test("2. 200: resolves with reviews sorted by date in descending order by default", () => {
+  return request(app)
+    .get('/api/reviews')
+    .expect(200)
+    .then(({ body }) => {
+      const { reviews } = body;
+      expect(reviews).toBeSortedBy('created_at',{descending:true})
+    });
+});
+
+test("3. 404: send a non-existent column to sort by", () => {
+  return request(app)
+    .get('/api/reviews?sort_by=vote; DROPTABLES')
+    .expect(404)
+    .then(({body:{msg}})=>{
+        expect(msg).toBe('No sort_by found');
+    });
+});
```

Comment:
The above would be a 400 as the sort by is completely invalid even though it is the correct data type